lib/eventFactory.js

```diff
@@ -453,50 +455,88 @@ async function convertStages(config) {
             }
         });
 
+        newStage.eventId = eventId;
         newStage.jobIds = jobIds;
+
         delete newStage.jobs; // extra field from yaml parser
 
+        // Check for setup and teardown
+        const setupJobName = `${STAGE_PREFIX}${key}:setup`;
+        const teardownJobName = `${STAGE_PREFIX}${key}:teardown`;
```

Comment:
Can the format for setup and teardown jobs be abstracted into to helper function?
```suggestion
        const setupJobName = getStageSetupJobName(key);
        const teardownJobName = getStageTeardownJobName(key);
```
Instead of hardcoding it in multiple places. Ex:

- https://github.com/screwdriver-cd/screwdriver/pull/2870/files#diff-e01d96efa0729764af51697e9dd42865efba14cb4562ebcb4207cb75528b4ffcR855
- https://github.com/screwdriver-cd/screwdriver/pull/2870/files#diff-e01d96efa0729764af51697e9dd42865efba14cb4562ebcb4207cb75528b4ffcR877

Reply:
sure